content/lxc/getting-started.md

```diff
@@ -443,7 +452,7 @@ You can then confirm its status with either of:
 
 And get a shell inside it with:
 
-    lxc-attach --name mycontainer
+    lxc-unpriv-attach --name mycontainer
```

Comment:
This is a distro-specific command, not an upstream one

Reply:
For distros without  `lxc-unpriv-attach` how are users supposed to attach unprivileged containers? [It fails with permission errors](https://github.com/lxc/lxc/issues/4440#issuecomment-2096585055)

Reply:
@stgraber would you care to answer so that I may fix this document properly?